.github/workflows/upload-api.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Setup
         uses: actions/setup-node@v3
         with:
-          node-version: 18
```

Comment:
I'd love to matrix test w/ both 16 and 18 if possible

Reply:
I'm not sure it is worth it, because doing it in ucanto increases test time significantly. If we ever find that something does not work in node 18 lets add then.